src/tronls.jl

```diff
@@ -1,3 +1,5 @@
+export TronSolverNLS
+
 const tronls_allowed_subsolvers = [:cgls, :crls, :lsqr, :lsmr]
```

Comment:
```suggestion
const tronls_allowed_subsolvers = [:cgls, :crls, :lslq, :lsqr, :lsmr]
```

Reply:
I think we should add (and probably test) this in a separate PR